include/sqlpp11/data_types/time_point/data_type.h

```diff
@@ -40,5 +40,9 @@ namespace sqlpp
     template <typename T>
     using _is_valid_operand = is_day_or_time_point_t<T>;
   };
+
+  using datetime = time_point;
+  using timestamp = time_point;
+  using time = time_point;
```

Comment:
datetime and timestamp are certainly OK, as long as everybody agrees on the timezone.

But are you sure about time? Doesn't that represent time of day?